src/Microsoft.ML.Recommender/MatrixFactorizationTrainer.cs

```diff
@@ -78,11 +78,13 @@ namespace Microsoft.ML.Trainers
     ///     </list>
     /// </p>
     /// <p>Example code can be found by searching for <i>MatrixFactorization</i> in <a href='https://github.com/dotnet/machinelearning'>ML.NET.</a></p>
+    /// <example>
     /// <format type="text/markdown">
     /// <![CDATA[
-    /// [!code-csharp[MF](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/MatrixFactorization.cs "A matrix factorization example.")]
```

Comment:
Why do we remove the example name? Why don't we use the entire file? Having 4 lines of copyright can free us from the risky line numbers.

Reply:
Yep, we are doing that with #1576. 

This PR was to show the documentation team what to fix for the 0.7 release.